src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs

```diff
@@ -102,6 +102,117 @@ public interface IAckableMessage
         int AckId { get; set; }
     }
 
+    /// <summary>
+    /// An access key request message.
+    /// </summary>
+    public class AccessKeyRequestMessage : ExtensibleServiceMessage
```

Comment:
How about put them in a new class?

Reply:
All other messages are in the same class, I assume it's a coding convention.

Reply:
question: is this one being used in rest api? If so, I need a tracing id (impl `IMessageWithTracingId`) for request logs.